src/js/components/receiving/ReceivingPage.jsx

```diff
@@ -29,6 +31,7 @@ class ReceivingPage extends Component {
       <PartialReceivingPage
         onSubmit={this.nextPage}
         shipmentId={this.props.match.params.shipmentId}
+        bins={this.state.bins}
```

Comment:
before the bins was saved in the PartialReceivingPage local state (not needed anywhere else), but since the candidates are fetched in the component (parent of PartialReceivingPage) I needed to move the fetch and pass the bins as a property to PartialReceivingPage

Reply:
Got it, thanks.